### PR TITLE
Added evolve and release push actions for pokemon

### PR DESCRIPTION
**Pull request by andythorne**
Added the ability to push `evolve_pokemon` and `release_pokemon` actions back to the bot.

See https://github.com/OpenPoGo/OpenPoGoBot/pull/117 for backend implementation.

Will look like this:
poke_actions

### CODE REVIEW

**Comment by MaxLeiter** [comment]
:+1: awesome

@Reaver01 : please be the one to merge this

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoWeb/pull-request/134/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoWeb)

**Comment by Reaver01** [comment]
Please resolve conflicts first

**Comment by kazyka** [comment]
:+1: 

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoWeb/pull-request/134/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoWeb)

**Comment by jadbalout** [comment]
Look into adding Power up aswell?

**Comment by andythorne** [comment]
Rebased :+1: 

@jadbalout There's no implementation of this in OpenPoGo yet. I'll add it in when it's implemented.

**Comment by wchill** [comment]
Rebased improperly; there's a remnant of a merge conflict in `main.js`.

File: js/main.js [hunk 1]
```diff
@@ -642,13 +695,53 @@ var mapView = {
       out += '<div class="col s12 m6 l3 center"><img src="image/pokemon/' +
         pkmnImage + '" class="png_img"><br><b>' +
         pkmnName +
+<<<<<<< bcb90dff8c4feaa8a5923443c321de268e455e64
```
Comment (wchill) [inline js/main.js]: Fix this

**Comment by andythorne** [comment]
Whoops! Fixed :)

**Comment by MaxLeiter** [comment]
LGTM :tada:

**Comment by eggins** [comment]
:+1:

**Pull request merged**